scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Deploys previously compiled contracts to provided network
+# Usage: ./scripts/deploy.sh NETWORK
```

Comment:
we have `bee` network, that I would rename to `docker` and I would invoke the `build-image.sh` from here and call `save-addresses` after the build like you do in other cases as well.